setup.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh -e
+
+OS=$(uname -s)
+ARCH=$(uname -m)
+LATEST_RELEASE_VERSION="0.9.1"
+
+installDocker() {
+  # Set default values
+  pguser="postgres"
```

Comment:
let's defaut pguser to `myuser` and pgpass to `mypassword`

Reply:
Cool! Added.